workflows/bacterial_genomics/bacterial_genome_assembly/bacterial_genome_assembly.ga

```diff
@@ -0,0 +1,705 @@
+            },
+            "post_job_actions": {
+                "RenameDatasetActionoutput_json": {
+                    "action_arguments": {
+                        "newname": "tooldistillator_results"
+                    },
+                    "action_type": "RenameDatasetAction",
+                    "output_name": "output_json"
+                },
+                "TagDatasetActionoutput_json": {
+                    "action_arguments": {
+                        "tags": "tooldistillator_results"
+                    },
+                    "action_type": "TagDatasetAction",
+                    "output_name": "output_json"
+                }
+            },
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/tooldistillator/tooldistillator/0.8.5.0+galaxy0",
+            "tool_shed_repository": {
+                "changeset_revision": "65a69de1e294",
+                "name": "tooldistillator",
+                "owner": "iuc",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"log\": false, \"tool_section\": {\"tools\": [{\"__index__\": 0, \"select_tool\": {\"tool_list\": \"shovill\", \"__current_case__\": 14, \"input\": {\"__class__\": \"ConnectedValue\"}, \"contig_graph_path\": {\"__class__\": \"ConnectedValue\"}, \"bam_file_path\": {\"__class__\": \"ConnectedValue\"}, \"analysis_software_version\": \"1.1.0\", \"reference_database_version\": null}}, {\"__index__\": 1, \"select_tool\": {\"tool_list\": \"quast\", \"__current_case__\": 11, \"input\": {\"__class__\": \"ConnectedValue\"}, \"quast_html_path\": {\"__class__\": \"ConnectedValue\"}, \"analysis_software_version\": \"5.2.0\", \"reference_database_version\": null}}, {\"__index__\": 2, \"select_tool\": {\"tool_list\": \"refseqmasher\", \"__current_case__\": 13, \"input\": {\"__class__\": \"ConnectedValue\"}, \"analysis_software_version\": \"0.1.2\", \"reference_database_version\": null}}, {\"__index__\": 3, \"select_tool\": {\"tool_list\": \"bandage\", \"__current_case__\": 3, \"input\": {\"__class__\": \"ConnectedValue\"}, \"bandage_plot_path\": {\"__class__\": \"ConnectedValue\"}, \"analysis_software_version\": \"0.8.1\", \"reference_database_version\": null}}]}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "0.8.5.0+galaxy0",
+            "type": "tool",
+            "uuid": "012cceba-b1d0-4f61-b0ed-762a776989a5",
+            "when": null,
+            "workflow_outputs": [
+                {
+                    "label": "tooldistillator_results",
+                    "output_name": "output_json",
+                    "uuid": "5699519c-5c9d-4247-bdf2-7732888f59bc"
+                }
+            ]
+        },
+        "8": {
+            "annotation": "ToolDistillator summarize groups all JSON file into a unique JSON file",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/tooldistillator_summarize/tooldistillator_summarize/0.8.5.0+galaxy0",
```

Comment:
Why is this step necessary when you're only producing a single json report in the first place?